questionpy_server/worker/runtime/messages.py

```diff
@@ -194,20 +194,36 @@ class ErrorType(IntEnum):
     type: ErrorType
     message: Optional[str]
 
+    original_stacktrace: Optional[str] = None
+    """The original worker-side stacktrace."""
+
     @classmethod
     def from_exception(cls, error: Exception, cause: MessageToWorker) -> "WorkerError":
         """Get a WorkerError message from an exception."""
         if isinstance(error, MemoryError):
             error_type = WorkerError.ErrorType.MEMORY_EXCEEDED
         else:
             error_type = WorkerError.ErrorType.UNKNOWN
-        return WorkerError(type=error_type, message=str(error), expected_response_id=cause.Response.message_id)
+
+        if __debug__:
+            original_stacktrace = "\n".join(traceback.format_exception(type(error), error, error.__traceback__))
```

Comment:
`format_exception` fügt selbst schon ein `\n` ans Ende der Strings.
```suggestion
            original_stacktrace = "".join(traceback.format_exception(type(error), error, error.__traceback__))
```